### ISSUE TITLE
Feature request (?) - Disable "Add Desklets" for non-admin users

### ISSUE THREAD

**Issue by MisterDiodes** [linuxmint/cinnamon-spices-desklets#813]
* Cinnamon version 5.2.x,  ~ 5.4
 * Distribution - Testing in Mint 20.3, 21
* Hardware - any 
**Issue**

This may be due to lack of knowledge on how to manage desklets for a workplace Linux machine, but we need a way to outright disable desklets / applets - or at least we need a way to HIDE the "Add Desklets" (or Add Applets) on the desktop / panel Alt-Click menu (for non-admin users).

The system seems to be password-protected everywhere for installing / removing / updating software, but the management system for desklets / applets seems wide open.  But maybe I'm missing the correct way to turn these features OFF for non-admin users.

This also applies to Configure Panel (Panel edit mode should require a password, as well as "Add Applets".

**Steps to reproduce**
It seems like any user can try to add Desklets / Applets etc.  Maybe I'm not seeing the way to disable these functions.

**Expected behaviour**
For a workplace PC, we really need to disable or hide the "Add Desklets" or "Applets" or be able to put the panel into "Edit mode" - at least these functions should be password protected.

**Other information**
Very happy to provide a decent donation to the team for helping get this figured out!

**Comment by koelle25** [comment]
I don't think that will be possible. AFAIK the spices (Desklets, Applets, etc.) all run completely in user space. They are being "installed" to `~/.local/share/cinnamon/applets` etc. and configurations reside in `~/.cinnamon/configs`.

As a workaround you could try (pre-creating and) `chown`ing these directories by root. That effectively prevents the user to make changes in these directories and therefore installation/configuration should not be possible anymore.

Regarding editing the panel I have no idea how you would be able to prevent this.
